nomic/tests/test_atlas_client.py

```diff
@@ -400,23 +364,20 @@ def test_map_text_pandas():
         'color': [random.choice(['red', 'blue', 'green']) for i in range(size)],
     })
 
-    project = atlas.map_text(
+    dataset = atlas.map_data(
         name='UNITTEST_pandas_text',
         id_field='id',
```

Comment:
We should test the ability to auto-generate ids in the pandas test--this has bit us before.

```suggestion
```